### PR TITLE
fix linting failures

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
This fixes some rubocop errors introduced with the rubocop-rspec gem update.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -104,20 +104,21 @@ Lint/NonDeterministicRequireOrder:
...
     - 'app/policies/evss_policy.rb'
     - 'spec/jobs/facilities/pssg_download_spec.rb'
+    - 'spec/request/authentication/standard_authentication_spec.rb'
     - 'spec/support/saml/response_builder.rb'
 
 # Offense count: 1
 Lint/StructNewOverride:
   Exclude:
-    - 'spec/request/authentication/standard_authentication.rb'
+    - 'spec/request/authentication/standard_authentication_spec.rb'
```
Comment (ericboehs) [inline .rubocop_todo.yml:121]: This line doesn't need to be here now. Right?

File: spec/request/authentication/standard_authentication_spec.rb [hunk 2]
```diff
@@ -3,7 +3,11 @@
 require 'rails_helper'
 
 # Note these specs MUST be run in order
+
+# rubocop:disable RSpec/FilePath
 RSpec.describe 'authenticating loa3 user', type: :request, order: :defined do
+  # rubocop:enable RSpec/FilePath
```
Comment (ericboehs) [inline spec/request/authentication/standard_authentication_spec.rb:9]: This also doesn't need to be here.

File: .rubocop_todo.yml (hunk 1 above)
Comment (LindseySaari) [reply .rubocop_todo.yml:121]: This is a different rubocop error. RSpec/FilePath was failing when _spec.rb wasn't at the end of the filename

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -104,20 +104,21 @@ Lint/NonDeterministicRequireOrder:
...
-# Offense count: 5
+# Offense count: 6
 # Cop supports --auto-correct.
 Lint/RedundantCopDisableDirective:
   Exclude:
     - 'app/models/emis_redis/military_information_v2.rb'
     - 'app/models/saved_claim/education_benefits/va_10203.rb'
     - 'app/policies/evss_policy.rb'
     - 'spec/jobs/facilities/pssg_download_spec.rb'
+    - 'spec/request/authentication/standard_authentication_spec.rb'
```
Comment (ericboehs) [inline .rubocop_todo.yml:115]: This shouldn't be needed either. 😬

File: .rubocop_todo.yml [hunk 4]
```diff
@@ -104,7 +104,7 @@ Lint/NonDeterministicRequireOrder:
     - 'spec/request/v1/apidoc/apidoc_spec.rb'
     - 'spec/service_consumers/pact_helper.rb'
 
-# Offense count: 5
+# Offense count: 6
```
Comment (ericboehs) [inline .rubocop_todo.yml:107]: The offense count is wrong but 👨‍🦯

**Review by ericboehs — APPROVED** [review]

File: spec/request/authentication/standard_authentication_spec.rb [hunk 5]
```diff
@@ -3,6 +3,7 @@
 require 'rails_helper'
 
 # Note these specs MUST be run in order
+
```
Comment (ericboehs) [inline spec/request/authentication/standard_authentication_spec.rb:6]:
```suggestion
```

File: .rubocop_todo.yml (hunk 4 above)
Comment (ericboehs) [inline .rubocop_todo.yml:107]:
```suggestion
# Offense count: 5
```

**Review by ericboehs — APPROVED** [review]
👨‍🍳🤌

**Pull request merged**